features/registry/registry.feature

```diff
@@ -106,6 +106,90 @@ Feature: Testing registry
     And the "myimage2" image stream was created
     And the "myimage1" image stream becomes ready
     And the "myimage2" image stream becomes ready
+    
+  # @author wewang@redhat.com
+  # @case_id OCP-23030
+  @admin
+  Scenario: Enable must-gather object refs in image-registry cluster
+    When I run the :get admin command with:
+      | resource      | co             |
+      | resource_name | image-registry |
+      | o             | yaml           |
+    Then the step should succeed
+    And the output should contain:
+      | name: system:registry                      |
+      | resource: clusterroles                     |
+      | name: registry-registry-role               |
+      | resource: clusterrolebindings              |
+      | name: registry                             |       
+      | resource: serviceaccounts                  |
+      | name: image-registry-certificates          |
+      | resource: configmaps                       |
+      | name: image-registry-private-configuration |
+      | resource: secrets                          | 
+      | name: image-registry                       |
+      | resource: services                         |
+      | name: node-ca                              |
+      | resource: daemonsets                       |
+      | name: image-registry                       |
+      | resource: deployments                      |
+    When I run the :get admin command with:
+      | resource      | clusterroles    |
+      | resource_name | system:registry |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | clusterrolebindings    |
+      | resource_name | registry-registry-role |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | serviceaccounts          |
+      | resource_name | registry                 |
+      | namespace     | openshift-image-registry |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | configmaps                  |
+      | resource_name | image-registry-certificates |
+      | namespace     |  openshift-image-registry   |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | secrets                              |
+      | resource_name | image-registry-private-configuration |
+      | namespace     |  openshift-image-registry            |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | ds                        |
+      | resource_name | node-ca                   |
+      | namespace     |  openshift-image-registry |
+    Then the step should succeed
+    When I run the :get admin command with:
+      | resource      | service                   |
+      | resource_name | image-registry            |
+      | namespace     |  openshift-image-registry |
```

Comment:
@wewang58 extra space in front of `openshift-image-registry`

Reply:
@pruan-rht Oh, I am sorry for that, Updated now